account_invoice_refund_link/readme/USAGE.rst

```diff
@@ -0,0 +1,5 @@
+This module creates in the invoice form, a new 'Refunds' page when the invoice
```

Comment:
Put steps as usual with `#.` numbered elements and showing where to go and so.

Reply:
done